source/robot_model/test/tests/test_kinematics.cpp

```diff
@@ -17,6 +17,7 @@ class RobotModelKinematicsTest : public testing::Test {
     urdf_path = std::string(TEST_FIXTURES) + "panda_arm.urdf";
     franka = std::make_unique<Model>(robot_name, urdf_path);
     joint_state = state_representation::JointState(robot_name, franka->get_joint_frames());
+    this->setTestConfigurations();
```

Comment:
Goodness, what a thing to miss. I guess because everything was value-initialised to zero it somehow all worked out to identity conditions.

While we're here, this library is using the `snake_case` naming convention, so the function should be renamed to `set_test_configurations()` (this will make it consistent with the test_dynamics file).

Reply:
(I copied this from you @eeberhard, look in the test_dynamics.cpp file :wink: )